specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -10056,7 +10059,7 @@
         ],
         "operationId": "ComputeNode_GetRemoteDesktop",
         "summary": "Gets the Remote Desktop Protocol file for the specified compute node.",
-        "description": "Before you can access a node by using the RDP file, you must create a user account on the node. This API can only be invoked on pools created with the cloud service configuration property.",
+        "description": "Before you can access a node by using the RDP file, you must create a user account on the node. This API can only be invoked on pools created with the cloud service configuration property. For pools created with a virtual machine configuration, see the GetRemoteLoginSettings API.",
```

Comment:
>cloud service configuration [](start = 175, length = 27)

"cloudServiceConfiguration" or "created in a cloud service configuration" (that is, if you are going to separate the words then don't say you're referring to the property)